lib/utils.js

```diff
@@ -31,6 +32,13 @@ function trimAllParents(paths, parents) {
   return paths.map(path => trimParents(path, parents))
 }
 
+function isValidExcludingGit(file) {
+  const exclusions = ['.git']
+  if (!file) return false
+  const segments = file.split(/[\\/]/g)
+  return !intersection(segments, exclusions).length
```

Comment:
To me, this would make the intention of this logic a little bit clearer than using the exclamation mark operator 🤔 
```suggestion
  return intersection(segments, exclusions).length === 0
```